modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/OpenAPIResourceTest.java

```diff
@@ -281,6 +329,18 @@ private void _assertOpenAPI(
 			JSONCompareMode.STRICT);
 	}
 
+	private void _assertOpenAPI(String fileName, String endpoint)
```

Comment:
You are invoking this method only once, and you are calling just one method, better to get rid of it